tests/e2e/testfiles/csm_authorization_proxy_server.yaml

```diff
@@ -0,0 +1,94 @@
+metadata:
+  name: authorization
+  namespace: authorization
+spec:
+  modules:
+    # Authorization: enable csm-authorization proxy server for RBAC
+    - name: authorization-proxy-server
+      # enable: Enable/Disable csm-authorization
+      enabled: true
+      configVersion: v1.4.0
+      forceRemoveModule: true
+      components:
+      - name: karavi-authorization-proxy-server
+        enabled: true
+        image: dellemc/csm-authorization-proxy:v1.4.0
+        image: dellemc/csm-authorization-tenant:v1.4.0
+        image: dellemc/csm-authorization-role:v1.4.0
+        image: dellemc/csm-authorization-storage:v1.4.0
+        image: redis:6.0.8-alpine
+        image: rediscommander/redis-commander:latest
+        image: openpolicyagent/opa
+        image: openpolicyagent/kube-mgmt:0.11
+        envs:
+          # base hostname for the ingress rules that expose the services
+          # the proxy-server ingress will use this hostname
+          # the tenant-service ingress will use tenant.hostname
+          # the role-service ingress will use role.hostname
+          # the storage-service ingress will use storage.hostname
+          # Allowed values: string
+          # Default value: csm-authorization.com
+          - name: "PROXY_HOST"
+            value: "csm-authorization.com"
+
+          # Authorization log level
+          # Allowed values: "error", "warn"/"warning", "info", "debug"
+          # Default value: "debug"
+          - name: "AUTHORIZATION_LOG_LEVEL"
+            value: "debug"
+
+          # Zipkin tracing configuration
+          # this can be updated on the fly via the csm-config-params configMap
+          # Default value: None 
+          - name: "AUTHORIZATION_ZIPKIN_COLLECTORURI"
+            value: ""          
+          # Default value: None 
+          - name: "AUTHORIZATION_ZIPKIN_PROBABILITY"
+            value: ""
+
+          # Proxy-service ingress configuration
+          # Default value: nginx
+          - name: "PROXY_INGRESS_CLASSNAME"
+            value: "nginx"
+          # Additional host rules for the proxy-server ingress
+          # Default value: authorization-ingress-nginx-controller.namespace.svc.cluster.local
+          - name: "PROXY_INGRESS_HOSTS"
+            value: "authorization-ingress-nginx-controller.authorization.svc.cluster.local"
+
+          # Tenant-service ingress configuration
+          # Default value: nginx
+          - name: "TENANT_INGRESS_CLASSNAME"
+            value: "nginx"
+
+          # Role-service ingress configuration
+          # Default value: nginx
+          - name: "ROLE_INGRESS_CLASSNAME"
+            value: "nginx" 
+
+          # Storage-service ingress configuration
+          # Default value: nginx
+          - name: "STORAGE_INGRESS_CLASSNAME"
+            value: "nginx" 
+
+          # Specify storage class for redis. Otherwise, default storage class is used
+          - name: "REDIS_STORAGE_CLASS"
+            value: "local"
```

Comment:
Is this storage class automatically created or are you making this a prereq to running the E2E test?

Reply:
This is pre-requisite